src/Bus/DeliveryChain/Middleware/Validation/MessageViolationListException.php

```diff
@@ -67,9 +67,9 @@ public function getErrors(): array
         foreach ($this->violations as $commandViolationException) {
             $fieldName = $commandViolationException->getField();
 
-            if (preg_match('/\[(\w+)\]/', $fieldName, $extraQuestionsMatch)) {
-                $rootFieldName = 'extraQuestions';
-                $mainFieldName = $extraQuestionsMatch[1];
+            if (preg_match('/(\w+)\[(\w+)\]/', $fieldName, $extraQuestionsMatch)) {
```

Comment:
Why is this variable named `extraQuestionsMatch`? As far as I can tell this has nothing to do with questions.